### PR TITLE
chore(facade): add facades for indexOf, lastIndexOf, findIndex.

### PR DESCRIPTION
**Pull request by jelbourn**
Need these for one of the components I'm working on.

### CODE REVIEW

File: modules/angular2/src/facade/collection.dart [hunk 1]
```diff
@@ -94,8 +94,19 @@ class ListWrapper {
   static bool contains(List m, k) => m.contains(k);
   static List map(list, fn(item)) => list.map(fn).toList();
   static List filter(List list, bool fn(item)) => list.where(fn).toList();
+  static indexOf(List list, value, [int startIndex = 0]) => list.indexOf(value, startIndex);
```
Comment (kevmoo) [inline modules/angular2/src/facade/collection.dart]: please add return type

File: modules/angular2/src/facade/collection.dart [hunk 2]
```diff
@@ -94,8 +94,19 @@ class ListWrapper {
...
+  static lastIndexOf(List list, value, [int startIndex = null]) =>
```
Comment (kevmoo) [inline modules/angular2/src/facade/collection.dart]: ditto

File: modules/angular2/src/facade/collection.dart [hunk 3]
```diff
@@ -94,8 +94,19 @@ class ListWrapper {
...
+      list.lastIndexOf(value, startIndex == null ? list.length : startIndex);
   static find(List list, bool fn(item)) =>
```
Comment (kevmoo) [inline modules/angular2/src/facade/collection.dart]: While you're here. :smile:

Comment (jelbourn) [inline modules/angular2/src/facade/collection.dart]: Is there a correct return type for `find`?

Comment (kevmoo) [inline modules/angular2/src/facade/collection.dart]: Find, no. Sorry.

**Pull request closed without merging**